src/content/updates/v10-migration/designers.mdx

```diff
@@ -0,0 +1,200 @@
+---
+title: v10 Migration
+tabs: ['Overview', 'Designers', 'Developers']
+---
+
+<AnchorLinks>
+
+- [IBM Design Language](#ibm-design-language)
+- [Design kit](#design-kit)
+- [Themes](#themes)
+- [Icons](#icons)
+- [Typography](#typography)
+- [Motion](#motion)
+- [Layout and grid](#layout-and-grid)
+- [Design migration strategy](#design-migration-strategy)
+<!-- - [Design FAQ](#design-faq) -->
+- [Feedback](#feedback)
+
+</AnchorLinks>
+
+## Resources
+
+<GridWrapper col_lg="8" flex="true" bleed="true">
```

Comment:
Add `className="tile--resource--no-margin"` to `GridWrapper` to remove spacing between tiles. If you want an example, check out the Resources page.

Reply:
Thanks Matt!